mediaengine.go

```diff
@@ -0,0 +1,206 @@
+package webrtc
+
+import (
+	"strconv"
+
+	"github.com/pions/webrtc/internal/sdp"
+	"github.com/pions/webrtc/pkg/rtp"
+	"github.com/pions/webrtc/pkg/rtp/codecs"
+	"github.com/pkg/errors"
+)
+
+// PayloadTypes for the default codecs
+const (
+	PayloadTypeOpus = 111
```

Comment:
Maybe indicate these payload types as "PayloadTypeOpusDefault" or something, as this really is an arbitrary value